src/components/elements/List/style.scss

```diff
@@ -0,0 +1,80 @@
+:local {
+  @import '../../../styles/config';
+
+  .List {
+    margin-top: 0;
+    margin-bottom: 0;
+    padding-left: 0;
+    list-style: none;
+
+    > li {
```

Comment:
OK, it looks like there are some limitation to use BEM styling in reactjs. However, for the site speed and maintenance, we should be careful about using generic element selectors and higher specificity. Let's ship it and think about this later. Here is one article I found and there can be more resources we can learn. :-D https://codepen.io/Merri/post/advanced-bem-with-react-components